duct/sources/linux/basic.py

```diff
@@ -262,10 +262,15 @@ class Memory(Source):
     def _parse_stats(self, mem):
         dat = {}
         for l in mem:
+            if not ':' in l:
```

Comment:
The preferred form is `if ':' not in l:` -- at least one linting tool would complain here.